verde/tests/test_base.py

```diff
@@ -18,8 +18,11 @@
 
 def test_get_dims():
     "Tests that get_dims returns the expected results"
-    assert get_dims(dims=None) == ("northing", "easting")
-    assert get_dims(dims=("john", "paul")) == ("john", "paul")
+    gridder = BaseGridder()
+    assert get_dims(gridder, dims=None) == ("northing", "easting")
+    assert get_dims(gridder, dims=("john", "paul")) == ("john", "paul")
+    gridder.dims = ("latitude", "longitude")
+    assert get_dims(gridder, dims=None) == ("latitude", "longitude")
```

Comment:
This test is a bit useless since it does the same thing as the doctest. Could probably get rid of it.

Reply:
Now `_get_dims` is a small private method, so I don't see the point of having an example for it. Wouldn't be better to remove the example and keep the tests?

Reply:
In the end, it doesn't matter that much. They both test the same thing and it's just a matter of having it as a doctest or regular test. The advantage of the doctest is that it serves as internal documentation.